src/gui/deletionconfirmationdlg.h

```diff
@@ -50,13 +52,17 @@ class DeletionConfirmationDlg : public QDialog, private Ui::confirmDeletionDlg {
     else
       label->setText(tr("Are you sure you want to delete these %1 torrents from the transfer list?", "Are you sure you want to delete these 5 torrents from the transfer list?").arg(QString::number(size)));
     // Icons
-    lbl_warn->setPixmap(GuiIconProvider::instance()->getIcon("dialog-warning").pixmap(lbl_warn->height()));
-    lbl_warn->setFixedWidth(lbl_warn->height());
+    const QSize iconSize = Utils::Gui::largeIconSize();
+    lbl_warn->setPixmap(GuiIconProvider::instance()->getIcon("dialog-warning").pixmap(iconSize));
+    lbl_warn->setFixedWidth(iconSize.width());
     rememberBtn->setIcon(GuiIconProvider::instance()->getIcon("object-locked"));
+    rememberBtn->setIconSize(Utils::Gui::mediumIconSize());
 
     checkPermDelete->setChecked(defaultDeleteFiles || Preferences::instance()->deleteTorrentFilesAsDefault());
     connect(checkPermDelete, SIGNAL(clicked()), this, SLOT(updateRememberButtonState()));
     buttonBox->button(QDialogButtonBox::Cancel)->setFocus();
+
+    resize(this->size() * Utils::Gui::screenScalingFactor(this));
```

Comment:
I would implement `Utils::Gui::resize(QWidget *)` (with some better/meaningful name, of course).

Reply:
Do you mean `Utils::Gui::screenScalingFactor` replaced by `Utils::Gui::resize(QWidget *)`?
Or add the latter as another function? I prefer adding it, not all `screenScalingFactor` usage have the same pattern.

Reply:
>Or add the latter as another function?

Yes.

Reply:
Done.